src/core/shaderlib/project/viewport-uniforms.js

```diff
@@ -124,14 +127,22 @@ export function getUniformsFromViewport({
     log.removed('positionOrigin', 'coordinateOrigin');
   }
 
+  return Object.assign(
+    {
+      project_uModelMatrix: modelMatrix || IDENTITY_MATRIX
+    },
+    memoized({viewport, coordinateSystem, coordinateOrigin})
+  );
+}
+
+function _getUniformsFromViewport({viewport, coordinateSystem, coordinateOrigin}) {
```

Comment:
Nit: We usually don't use underscores for functions. The lack of the `export` keyword is enough to show that a function is internal. The official function is usually just called `getUniforms` in most modules if that helps. Since it only covers some uniforms, maybe call it `getCoreUniforms` or `getMemoizableUniforms`?